examples/index.html

```diff
@@ -116,6 +116,26 @@ <h1>Share</h1>
               placeholder="Title"
             />
           </div>
+          <div class="modal fade" id="myModal" role="dialog">
+            <div class="modal-dialog">
+              <!-- Modal content-->
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h4 class="modal-title">Modal Header</h4>
+                </div>
+                <div class="modal-body">
+                  <p>First, a good title predicts the content of the research paper. 
```

Comment:
Maybe listing the good title requirements as a list would be better and more memorable to the user than plain text.
What do you think?